Dockerfile

```diff
@@ -34,9 +34,8 @@ LABEL org.opencontainers.image.title="${SERVICE_NAME}" \
       org.opencontainers.image.revision="${COMMIT}" \
       org.opencontainers.image.created="${BUILD_DATE}"
 
-RUN dnf install -y libpq-15.0
-
-RUN mkdir -p /var/www/${SERVICE_NAME}/ \
+RUN dnf install -y libpq-15.0 && \
```

Comment:
Formatting is inconsistent here. The '&&' on the first line should be on the second to match the 3rd.